internal/wazeroir/operations_test.go

```diff
@@ -19,8 +19,14 @@ func TestUnionOperation_String(t *testing.T) {
 	for _, k := range []OperationKind{
```

Comment:
```suggestion
	// TODO: after done with union refactoring, move `for k := OperationKind(0); k < operationKindEnd; k++ { require.NotEqual(t, "", k.String())}` rather than listing all kinds here manually..
	for _, k := range []OperationKind{
```

Reply:
I noticed you missed adding in the list, so I added some comments for the future 👍 @evacchi